pkg/virt-controller/watch/vmi.go

```diff
@@ -380,6 +380,42 @@ func (c *VMIController) updateStatus(vmi *virtv1.VirtualMachineInstance, pod *k8
 			conditionManager.RemoveCondition(vmiCopy, virtv1.VirtualMachineInstanceConditionType(k8sv1.PodReady))
 		}
 
+		var networkPatchNeeded bool
+		// IF the VMI has a pod network, Check and patch if IP on VMI is not same as the one on Pod IP
+		for _, network := range vmiCopy.Spec.Networks {
```

Comment:
I know we're only looking for a single pod network address, but 3 `for`s... I would at least break the network lookup to a variable. (or a map if you expect more than one pod network)

Reply:
There is only a single Pod network. I think we are safe counting on that.

Reply:
@vladikr Yes it's not a big look but good suggestion to improve it, moved the network lookup to a map